tools/walletextension/wallet_extension.go

```diff
@@ -412,6 +420,11 @@ func (we *WalletExtension) handleSubmitViewingKey(userConn userconn.UserConn) {
 	we.accountManager.AddClient(accAddress, client)
 
 	we.persistence.PersistViewingKey(vk)
```

Comment:
this line, together with the old "persistence" need to be removed.

Reply:
Persistence removed.